File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
    int count=0,i,j;
    for(i=0;i<nums.size();i++)
    {
        for(j=i+1;j<nums.size();j++)
        {
            if(nums[i]+nums[j]==target)
           {
               count++;
               break;
               
           }
        }
        if(count==1)
        {
            break;
        }  
    }  
         return {i,j};      
    }
};

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> freq= new HashMap<>();
        int[] res = new int[2];
		for(int i = 0 ; i < nums.length ; i++)
		{
            if(freq.containsKey(target - nums[i]))
			{
                res[0] = i;
                res[1] = freq.get(target - nums[i]);
                break;
			}
			freq.put( nums[i] , i );
		}
        return res;
    }
}